src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6133,4 +6133,19 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
   <data name="InstallPackageMessage" xml:space="preserve">
     <value>Installing package '{0}' to '{1}'.</value>
   </data>
+  <data name="Error_ResponseFileDoesNotExist" xml:space="preserve">
+    <value>error NG5002: Response file '{0}' does not exist</value>
+  </data>
+  <data name="Error_ResponseFileInvalid" xml:space="preserve">
+    <value>error NG5001: Invalid response file, '@' does not exist</value>
+  </data>
+  <data name="Error_ResponseFileMaxRecursionDepth" xml:space="preserve">
+    <value>error NG5005: No more than {0} nested response files are allowed</value>
+  </data>
+  <data name="Error_ResponseFileNullOrEmpty" xml:space="preserve">
+    <value>error NG5003: Response file '{0}' cannot be empty</value>
+  </data>
+  <data name="Error_ResponseFileTooLarge" xml:space="preserve">
+    <value>error NG5004: Response file '{0}' cannot be larger than {1}mb</value>
```

Comment:
These codes should be sent through the normal logging interface, not as part of the localized strings. We should also re-evaluate the number ranges used here, and keep it in sync with the rest of the codes. Restore uses `NU` for example so having `NG` here is confusing.

//cc @mishra14

Reply:
So you want something like the restore command where it concats the messages to a code and puts all the logging messages thru another method?

Reply:
There are multiple loggers used in the project, can you point me to a code example of one so I can use the correct one?

Reply:
Log using `ILogger.Log`
https://github.com/NuGet/NuGet.Client/blob/710ec89011b58a78a2bbff3a88771fbd4606db6b/src/NuGet.Core/NuGet.Common/Logging/ILogger.cs#L32

You can log a `LogMessage`
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/Errors/LogMessage.cs

If you need to extend `LogMessage` into `PackageLogMessage` similar to `RestoreLogMessage`

Logging for nuget.exe to the console happens here: 

https://github.com/NuGet/NuGet.Client/blob/710ec89011b58a78a2bbff3a88771fbd4606db6b/src/NuGet.Clients/NuGet.CommandLine/Common/Console.cs#L384-L406

At that point you can convert the message into the expected format if it differs from the default

Reply:
Perfect, thank you.  Will update.

Reply:
You can add the new codes into https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

I suggest keeping everything in the 5000-6000 range for pack

Reply:
Will update codes, I was using the "pack" range defined in the enum comment 1400-1900

Reply:
Updated logging to use simlified RestoreLogMessage pattern